packages/pangea-multipass-langchain/pyproject.toml

```diff
@@ -1,20 +1,23 @@
-[tool.poetry]
+[project]
 name = "pangea-multipass-langchain"
 version = "0.2.0"
 description = "Pangea Multipass authorization library for LangChain"
-authors = ["Apurv Jawle <apurv.jawle@pangea.cloud>"]
+authors = [
+    { name = "Apurv Jawle", email = "apurv.jawle@pangea.cloud" }
+]
 license = "MIT"
 readme = "README.md"
+requires-python = ">=3.10,<4.0"
+dependencies = [
+    "langchain (==0.3.18)",
+]
```

Comment:
Are you sure you don't want pangea-multipass here? I know it's below as a relative path, but that's only for local development; for downstream users I do think the package needs to have pangea-multipass in its `dependencies`.

Reply:
I was planning to add it here just when doing a release, not sure exactly how it handle relatives paths on build tests that maybe refer to latests modules in the package. Maybe I should add it without version?

Reply:
Hmm I think there's a misunderstanding here. I believe you should be able to have `pangea-multipass (>=0.2.0)` here at all times. When you publish a new pangea-multipass version, you can then bump to that new version number in `-langchain` and `-llama-index`'s dependencies list if necessary. Then you publish those two packages (if they have any changes). Less work than you needing to add this for a release and then presumably remove it afterwards. The relative path only has any affect on folks that are working from this repository itself.

Reply:
Okey, make sense, I've updated it now.